phpBB/phpbb/textformatter/s9e/factory.php

```diff
@@ -84,6 +84,10 @@ class factory implements \phpbb\textformatter\cache_interface
 		'img'   => '[IMG src={IMAGEURL;useContent}]',
 		'list'  => '[LIST type={HASHMAP=1:decimal,a:lower-alpha,A:upper-alpha,i:lower-roman,I:upper-roman;optional;postFilter=#simpletext} #createChild=LI]{TEXT}[/LIST]',
 		'li'    => '[* $tagName=LI]{TEXT}[/*]',
+		'mention' =>
+			"[MENTION={PARSE=/^(?<type>[ug]):(?<id>\d+)$/}
+				profile_url={URL;optional;postFilter=#false}
```

Comment:
It would be a good idea to define the `color` attribute here (e.g. `color={COLOR}`) otherwise the template is potentially unsafe since it's used as-is in CSS.

Reply:
Added color attribute.
@JoshyPHP BTW could you please explain the meaning/necessity of `postFilter=#false`?
I've taken that from the `quote` BBCode.

Reply:
[It unconditionally invalidates the value](https://s9etextformatter.readthedocs.io/Filters/Built-in_filters/). That prevents the user from supplying a value for the profile URL.

Reply:
OK, so my guesses were right, thank you.